assets/src/edit-story/app/rightClickMenu/provider.js

```diff
@@ -193,6 +278,88 @@ function RightClickMenuProvider({ children }) {
     [handleCopyPage, menuItemProps, handleDeletePage, handlePastePage]
   );
 
+  const foregroundMediaItems = useMemo(
+    () => [
+      ...defaultItems,
+      {
+        label: RIGHT_CLICK_MENU_LABELS.SEND_BACKWARD,
+        separator: 'top',
+        // TODO: this shortcut does not exist yet. Add shortcut to editor.
+        shortcut: {
+          display: isMacOs ? '⌥ ⌘ [' : '⌥ ctrl [',
+          title: isMacOs
+            ? RIGHT_CLICK_MENU_SHORTCUT_LABELS.OPTION_COMMAND_OPEN_BRACKET
+            : RIGHT_CLICK_MENU_SHORTCUT_LABELS.OPTION_CONTROL_OPEN_BRACKET,
+        },
+        disabled: !canElementMoveBackwards,
+        onClick: handleSendBackward,
+        ...menuItemProps,
+      },
+      {
+        label: RIGHT_CLICK_MENU_LABELS.SEND_TO_BACK,
+        // TODO: this shortcut does not exist yet. Add shortcut to editor.
```

Comment:
None of the 'send forward, backward' buttons have shortcuts attached in the editor yet.

Reply:
@BrittanyIRL might have more knowledge here, but thanks for flagging!

Reply:
ticket here: https://github.com/google/web-stories-wp/issues/8440  - if you get a second, can you add the ticket number to these `todo` comments?